webapp/src/Controller/Jury/ImportExportController.php

```diff
@@ -334,7 +343,7 @@ public function exportHtmlAction(Request $request, string $type): Response
         }
     }
 
-    protected function getResultsHtml(Request $request): Response
+    protected function getResultsHtml(Request $request, bool $full = false): Response
```

Comment:
I would consider replacing `$full` (not only here) with another variable with the opposite meaning, e.g. `$onlyMedals = false`.

Reply:
But it’s not only medals. It’s more “same rank for same number of problems”. Suggestions?

Reply:
meh, then leave it like it is now